vizro-core/examples/_chart-gallery/pages/magnitude.py

```diff
@@ -125,4 +158,36 @@
     ],
 )
 
-pages = [bar, column, paired_bar, paired_column]
+parallel_coordinates = vm.Page(
+    path="magnitude/parallel-coordinates ",
+    title="Parallel coordinates",
+    layout=vm.Layout(grid=PAGE_GRID),
+    components=[
+        vm.Card(
+            text="""
+                #### What is a parallel coordinates chart?
+
+                A parallel coordinates chart is a type of data visualization used to plot multivariate numerical data.
+                Each axis represents a different variable, and lines connecting the axes indicate the values of
+                individual data points across these variables.
+
+                &nbsp;
+
+                #### When should I use it?
+
+                Use a parallel coordinates chart to explore relationships and patterns in high-dimensional data.
+                This chart is particularly useful for comparing multiple variables simultaneously and identifying
+                correlations or clusters within the data. Ensure clear labeling of each axis and consider using color
+                coding to distinguish between different data points or groups.
+        """
+        ),
+        vm.Graph(
+            figure=px.parallel_coordinates(
+                iris, color="species_id", dimensions=["sepal_width", "sepal_length", "petal_width", "petal_length"]
```

Comment:
It doesn't seem like this is possible with Plotly - it only allows for a continuous color scale here: https://plotly.com/python-api-reference/generated/plotly.express.parallel_coordinates.html

Another idea would be to map integer values to "species," but the numbers 1, 2, and 3 would still be automatically visualized on a continuous scale.

The other idea is removing the color argument altogether (fine for me). Regarding the contrast, I've noticed that as well, but I couldn't figure out how to change the color of the lines in the back. It's not the grid-lines, nor any of the ones we define in our template, otherwise it would have changed 😅

Reply:
Hmm yes, doesn't seem to be possible indeed: https://github.com/plotly/plotly.py/issues/2494.

I think the color argument is still useful here because it helps you see there's a difference between the three species. So I'd just leave it as it is.